menu/cbs/menu_cbs_get_value.c

```diff
@@ -1098,6 +1098,34 @@ static void menu_action_setting_disp_set_label_playlist_associations(file_list_t
       strlcpy(s, msg_hash_to_str(MENU_ENUM_LABEL_VALUE_NOT_AVAILABLE), len);
 }
 
+static void menu_action_setting_disp_set_label_playlist_label_display_mode(
+      file_list_t* list,
+      unsigned *w, unsigned type, unsigned i,
+      const char *label,
+      char *s, size_t len,
+      const char *path,
+      char *s2, size_t len2)
+{
+   enum playlist_label_display_mode label_display_mode;
+   int msg_index;
+   playlist_t *playlist  = playlist_get_cached();
+
+   if (!playlist)
+      return;
+
+   label_display_mode = playlist_get_label_display_mode(playlist);
+
+   *w = 19;
+
+   strlcpy(s2, path, len2);
+
+   msg_index = (int)label_display_mode;
+   msg_index = msg_index * 3;
+   msg_index = msg_index + (int)MENU_ENUM_LABEL_VALUE_PLAYLIST_MANAGER_LABEL_DISPLAY_MODE_DEFAULT;
```

Comment:
Okay - this works, but it's not safe! It assumes that `#define MENU_LABEL(STR)` in `msg_hash.h` will never change, which is probably the case, but it's not guaranteed.

There needs to be a well-defined conversion here from `playlist_label_display_mode` to `msg_hash_enums` (I guess a `switch` would be most appropriate)